src/include/storage/storage_structure/disk_overflow_file.h

```diff
@@ -37,19 +37,24 @@ class DiskOverflowFile : public StorageStructure {
         return copy;
     }
 
-    void readStringsToVector(
-        transaction::TransactionType trxType, common::ValueVector& valueVector);
-    void readStringToVector(transaction::TransactionType trxType, common::ku_string_t& kuStr,
+    struct OverflowPageCache {
```

Comment:
Can you move this struct to private? I guess it doesn't need to be exposed as public for now.

Reply:
Hi! Thank you so much for your comments.

1. You are right, I actually created a csv with a very long string, but after querying, I realized it was stored only 4096 bytes.
2. I'll benchmark it soon. I believe the numbers should be better, considering the cache is implemented for the child type STRING (unless the object creating is causing some overhead, as mentioned by @andyfengHKU)